cognite/neat/rules/importers/_inference2rules.py

```diff
@@ -262,12 +275,9 @@ def _update_value_type_occurrence_in_comment(cls, value_type: str, comment: str)
 
     @classmethod
     def _read_value_type_occurrence_from_comment(cls, value_type: str, comment: str) -> int:
```

Comment:
Should be a task in our board to track occurence not as in comment but some private variable as regex is expensive